wgpu/gui/glfw.py

```diff
@@ -79,31 +55,71 @@ def __init__(self, *, size=None, title=None):
             if "wayland" in os.getenv("XDG_SESSION_TYPE", "").lower():
                 glfw.window_hint(glfw.FOCUSED, False)  # prevent Wayland focus error
 
-        self._window = glfw.create_window(width, height, title, None, None)
-        glfw.set_window_refresh_callback(self._window, self._paint)
+        # Create the window (the initial size may not be in logical pixels)
+        self._window = glfw.create_window(int(size[0]), int(size[1]), title, None, None)
+
+        # Register callbacks
+        glfw.set_window_content_scale_callback(self._window, self._on_pixelratio_change)
+        glfw.set_framebuffer_size_callback(self._window, self._on_size_change)
+        glfw.set_window_refresh_callback(self._window, self._on_paint)
```

Comment:
I have some concerns here; you can't guarantee that the application can keep up with the desired refresh rate. But either way, it's probably fine for now

Reply:
Yeah, let's revisit when we work on #63